README.md

```diff
@@ -35,6 +35,8 @@ const firstAction = () => {
 }
 ```
 
+You can use [redux-promise-middleware-actions](docs/guides/redux-promise-middleware-actions.md) to create async actions and handle them in your reducer.
```

Comment:
Let's not include a plug in the main README. I believe it would make the document too complicated. This README should just provide a brief overview, a table of contents, a few commonly used links and the license.